gnome-common-3.14 install ax_check_enable_debug.m4 and ax_code_coverage.m4
only when autoconf-archive is not installed
(https://bugzilla.gnome.org/show_bug.cgi?id=747920)
diff -Naur gnome-common-3.14.0.orig/configure.ac gnome-common-3.14.0/configure.ac
--- gnome-common-3.14.0.orig/configure.ac	2014-09-22 04:32:49.000000000 -0700
+++ gnome-common-3.14.0/configure.ac	2019-02-05 11:51:58.000000000 -0800
@@ -13,6 +13,14 @@
 fi
 AM_MAINTAINER_MODE([enable])
 
+AC_ARG_WITH([autoconf-archive],
+            [AS_HELP_STRING([--with-autoconf-archive],
+                            [Enable parallel-installability with autoconf-archive, by disabling installation of its M4 macros])],
+            [], [with_autoconf_archive=no])
+
+AM_CONDITIONAL([WITH_AUTOCONF_ARCHIVE],
+               [test "x$with_autoconf_archive" != "xno"])
+
 AC_CONFIG_FILES([
 Makefile
 macros2/Makefile
diff -Naur gnome-common-3.14.0.orig/macros2/Makefile.am gnome-common-3.14.0/macros2/Makefile.am
--- gnome-common-3.14.0.orig/macros2/Makefile.am	2014-08-29 09:28:20.000000000 -0700
+++ gnome-common-3.14.0/macros2/Makefile.am	2019-02-05 11:53:13.000000000 -0800
@@ -10,10 +10,13 @@
 # these should be dropped as soon as the deprecated GNOME2_MACROS which use them
 # are removed. Until then, they should be kept up to date, and all changes to
 # them submitted upstream.
+if !WITH_AUTOCONF_ARCHIVE
+# Install the macros only if autoconf-archive support is disabled.
 AUTOCONF_ARCHIVE_MACROS = \
 	ax_check_enable_debug.m4 \
 	ax_code_coverage.m4 \
 	$(NULL)
+endif
 
 EXTRA_DIST = $(GNOME2_MACROS) $(AUTOCONF_ARCHIVE_MACROS) gnome-autogen.sh
 
